README.md

```diff
@@ -12,7 +12,7 @@ curl http://127.0.0.1:8080/predictions/bert -T input.txt
 ```
```

Comment:
The README for `binaries/README.md` should be updated with instructions to `pip install requirements/developer.txt`

Reply:
This is being done by python ts_scripts/install_dependencies.py --environment=dev. Isn't that right?

Reply:
Oh yeah good point nvm